tests/test__influx_reporter.py

```diff
@@ -47,8 +47,7 @@ def test_gauge_without_tags(self):
             autocreate_database=False
         )
 
-        with mock.patch.object(influx_reporter, "_try_send",
-                               wraps=influx_reporter._try_send) as send_mock:
+        with mock.patch.object(influx_reporter, "_try_send") as send_mock:
```

Comment:
how is that related to the not report on blanks ?

Reply:
It's not, I can't track down why the test broke now, but it's not related to this pr. Probably some version difference between python 3.7 or anything specific to my environment(this project doesn't have a virtualenv).

Anyway I don't see the purpose of that wraps=, the test is not suppose to actually contact a real server, so it shouldn't be there.